offer.md

```diff
@@ -0,0 +1,13 @@
+# Looking to integrate a big feature?
+
+Check out [Add Features](integrators.md) for guidance on adding a large feature yourself.
+
+#### Bounty Instructions for Big Features:
+
+Projects are welcome and encouraged to offer bounties to get their feature integrated by the large and growing community of ShapeShift developers. The product and engineering workstreams have processes and resources defined to provide guidance, support, and code and UX reviews to any individuals or teams who take on a bounty.
```

Comment:
I like to use sentence case when referring to workstreams, i.e. `Product and Engineering workstreams`, as they are nouns.